libnmstate/schema.py

```diff
@@ -200,6 +200,14 @@ class Ethernet(object):
     FULL_DUPLEX = 'full'
     HALF_DUPLEX = 'half'
 
+    SR_IOV = 'sr-iov'
+
+
+class SRIOV(object):
+    CONFIG_SUBTREE = 'sr-iov'
```

Comment:
You can re-use the `Ethernet.SR_IOV` constant here. Can you please also explain in the commit message why it is a subtree of Ethernet?

Reply:
Done! The explanation added is: `SR-IOV is a subtree of ethernet because it contains the PF information which must be an ethernet interface.`